core/history_api.php

```diff
@@ -203,7 +203,7 @@ function history_get_range_result_filter( $p_filter, $p_start_time = null, $p_en
 		return db_empty_result();
 	}
 
-	$t_select_string = 'SELECT DISTINCT {bug}.id ';
+	$t_select_string = 'SELECT {bug}.id ';
```

Comment:
> do we get better performance without the DISTINCT ?

if you mean
SELECT .. WHERE bug_id IN (SELECT {bug}.id from... )
against
SELECT .. WHERE bug_id IN (SELECT **DISTINCT** {bug}.id from... )

oracle, mysql, plans are identical, so its already detecting the semijoin and optimizing.
postgres, reports doing an inner sort-unique that adds to cost and time.